### PR TITLE
Kai bot deploy

### PR DESCRIPTION
**Pull request by alexgraham-da**

### CODE REVIEW

File: ui/src/components/Investor/Investor.tsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ const Investor: React.FC<Props> = ({ onLogout }) => {
     const ledger = useLedger();
     const loading = usePartyLoading();
 
+
```
Comment (Sarahb-da) [inline ui/src/components/Investor/Investor.tsx:50]: added line

File: ui/src/components/SetupRequired.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    const timer = setInterval(() => {
+      getPublicAutomation(publicParty).then(autos => {
+        setAutomations(autos);
+        if (!!automations && automations.length > 0) {
+          clearInterval(timer);
+        }
+      });
+    }, 3000);
+    return () => clearInterval(timer);
+  }, [publicParty]);
```
Comment (Sarahb-da) [inline ui/src/components/SetupRequired.tsx:29]: Mind explaining what this is doing? The timer inside the hook? I dont totally follow

File: ui/src/components/SetupRequired.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      const args = {
+        operator: parties.userAdminParty,
+        public: parties.publicParty
+      }
+      await ledger.create(Operator, args);
+    } catch(e) {
+      console.log("error exercising setup: " + e);
+      setSetupError(true);
+    }
+
```
Comment (Sarahb-da) [inline ui/src/components/SetupRequired.tsx:47]: extra line

File: ui/src/components/SetupRequired.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        deployTrigger(TRIGGER_HASH, MarketplaceTrigger.OperatorTrigger, adminJwt, publicParty);
+      }
+      const args = {
+        operator: parties.userAdminParty,
+        public: parties.publicParty
+      }
+      await ledger.create(Operator, args);
+    } catch(e) {
+      console.log("error exercising setup: " + e);
+      setSetupError(true);
```
Comment (Sarahb-da) [inline ui/src/components/SetupRequired.tsx:45]: if the error could be somewhat specific u can make `setupError` a `string | undefined` and set it here, display it below

**Review by Sarahb-da — APPROVED** [review]
lgtm, minor comment or 2

File: ui/src/components/SetupRequired.tsx (hunk 2 above)
Comment (alexgraham-da) [reply ui/src/components/SetupRequired.tsx:29]: sure - I wanted the screen to change once the automation was made deployable, so this hits the deployable endpoint every 3 seconds until it returns with automations. If you know a better way of doing this, please let me know!

File: ui/src/components/SetupRequired.tsx (hunk 4 above)
Comment (alexgraham-da) [reply ui/src/components/SetupRequired.tsx:45]: nice 👍

**Pull request merged**